src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -622,6 +634,8 @@ export default {
 
 		// Load all available facets for specified sector
 		await this.fetchFacets();
+
+		this.backupLoans = this.loans.slice(3);
```

Comment:
I tried this on /lend-by-category/agriculture, and I didn't see the backup loans:

![image](https://user-images.githubusercontent.com/16867161/223229481-5fabbc67-5953-4e41-9b28-ac6d5903b104.png)

Reply:
Nice catch!
Do we wanna show promo card for this?

Reply:
Probably just the 3 loans...